### ISSUE TITLE
Error On Installation

### ISSUE THREAD

**Issue by tgd1973** [AzuraCast/AzuraCast#2035]
Hi I have an error in the traditional installation, not Docker, of Azuracast on a VPS in ESXi Vmware:
mkdir -p /var/azuracast/www
cd /var/azuracast/www
git clone https://github.com/AzuraCast/AzuraCast.git .
chmod a+x install.sh
./install.sh

The browser page marks HTTP ERROR 500  on end of install.
I tried to install in Ubuntu 16.04 and 18.04 and the error is the same:

TASK [azuracast-setup : Run AzuraCast Setup (Install Mode)] ********************
fatal: [localhost]: FAILED! => {"changed": true, "cmd": "php /var/azuracast/www/bin/azuracast.php azuracast:setup", "delta": "0:00:00.999545", "end": "2019-10-05 02:25:58.043372", "msg": "non-zero return code", "rc": 255, "start": "2019-10-05 02:25:57.043827", "stderr": "", "stderr_lines": [], "stdout": "", "stdout_lines": []}

RUNNING HANDLER [azuracast-user : send user notification] **********************

RUNNING HANDLER [azuracast-db-install : send mysql notification] ***************

PLAY RECAP *********************************************************************
localhost        : ok=87   changed=69   unreachable=0    failed=1    skipped=8    rescued=0    ignored=2

**Comment by ogrenci01** [comment]
Hello Everyone;

This problem has been reported many times. This is a known issue.
Here : #1930 , #1862 , #1848 , #1842

I want to help you with an immediate solution.

1-) `sudo apt-get update -y`
2-) `sudo apt-get install -q -y git`
3-) `sudo mkdir -p /var/azuracast/www`
4-) `cd /var/azuracast/www`
5-) `sudo git clone https://github.com/AzuraCast/AzuraCast.git .`
6-) `sudo chmod a+x install.sh`
7-) `sudo chmod a+x update.sh`
8-) Download an older version:
**https://github.com/azuracast/azuracast/archive/0.9.6.zip**
9-) Use FTP to upload this archive to the directory "/var/azuracast/www". Accept changes.
10-) `sudo ./install.sh` After completing the installation, start and complete the installation.
11-) Download the latest version after completing the installation.
**https://codeload.github.com/AzuraCast/AzuraCast/zip/master**
12-) Upload this archive to FTP directory "/var/azuracast/www". Accept changes.
13-) `sudo ./update.sh` After the installation is complete, start and complete the update.
14-) After the update is completed, Azuracast will be ready to use.

Enjoy!

**Comment by tgd1973** [comment]
Hello again! A little longer these 14 installation steps, but in the end it works.
Those who manage the software need to see what changes in the installation of the previous version compared to the last one which generates an installation error. 
Not everyone wants to install via docker.
Thanks for help

**Comment by SlvrEagle23** [comment]
@tgd1973 Hello, application creator and maintainer here.

As you may have noticed from a number of locations in our documentation, our Ansible/"Traditional" installation is not officially supported by our team. We intend these warnings to strongly discourage the average user from using this installation type, because of the extremely high potential for conflicts with other software installed on the same server, and the level of expertise required to resolve some of these issues. Despite these warnings, our Ansible installations still make up a disproportionately high share of our support burden, which makes life very difficult for us as we are a very small team and already quite occupied with other things.

I will look to see if there is something that we can do to resolve the issue with traditional installations not working out of the box currently, but please be aware that these installation types are not supported by our team in a majority of cases.

**Comment by ogrenci01** [comment]
Hello;

Azuracast is one of the best software ever made. It is also completely free. I appreciate the Azuracast team.

I am using Ansible and will use it in the future. I installed it with a clean Ubuntu. I just installed Azuracast. That's why I never had a problem. Some users want to use it with other software. So they see a lot of mistakes.

Installation with 14 options is not a problem for me.

Please do not leave support for Ansible even if it is small.

Thank you everybody.

**Comment by SlvrEagle23** [comment]
@ogrenci01 The Ansible/Traditional installation has been unsupported for months now. Our documentation has noted this for quite some time, and clearly deprioritizes the Ansible installation method for the Docker one. We don't guarantee that any new features will apply to Ansible installations and several, like the built-in FTP server, don't.

At this point, the Ansible installation should _only_ be used by two groups:
- Linux administration experts who know exactly what they're doing when customizing their servers, and are prepared to handle any compatibility issues and, in the event there is an error we can fix (like this one), send us detailed logs that identify the source of the problem so we can easily fix it, and
- Users who absolutely cannot run our Docker version on their server, whether due to their web hosts not supporting LXC/Docker or running on alternate hardware, like a Raspberry Pi 3/4.

**Comment by tgd1973** [comment]
I thank you all for the effort you are making. I'm coming from AirTime / Libretime which does not allow multiple instances and I then went to Mscp-Pro then I found you by chance.
I wish you go ahead, because your software is very good ... Characterized for a simple and clean interface. Perhaps that still to work on the schedule playlist because it is not clear enough that for each new playlist programmed at a certain time, you must also restart liquidsoap to be able to read the new playlist but this also blows up the song that is in progress.

However for me it seems more normal the classical way through Ansible instead of the docker containers that if anything happens they are like a closed box to which one cannot access with high ease.

Very good for now.
